api.swagger.yaml

```diff
@@ -7263,6 +7263,36 @@ paths:
         - SessionCookieHeader: []
         - APIKeyHeader: []
         - CognitoJwt: [ invotra/job_roles:write ]
+  '/job_roles/merge':
```

Comment:
I think we should do:

PUT ​/job_roles​/{job_role_id}/merge
body { merge_uuids[] }